include/LinearSolverConfig.h

```diff
@@ -59,6 +59,12 @@ class LinearSolverConfig
   inline bool useSegregatedSolver() const
   { return useSegregatedSolver_; }
 
+  inline bool simpleHypreMatrixAssemble() const
+  { return simpleHypreMatrixAssemble_; }
+
+  inline bool dumpHypreMatrixStats() const
+  { return dumpHypreMatrixStats_; }
+
```

Comment:
Seems like inheritance is breaking down here. Exposing subclass logic in base class?

Reply:
You're right. I should try to fix this.

Reply:
Fixed in the most recent commit.